.travis.yml

```diff
@@ -4,10 +4,12 @@ jdk:
 cache:
   directories:
   - "$HOME/.m2"
+install:
```

Comment:
What's the point of this? The travis `install` phase is meant for installing dependencies of this build. This does not seem like it here.
What we had before in the script phase seems legit (may be bumped up to `package`). Clean should not be necessary

Reply:
What we had before was the default which doesn't work (whatever it is) with snapshots or milestone dependencies. This works. If you find another incantation that also works feel free to change it.

Reply:
Isn't the addition of `-P psring` in the `script` section (line 12 below) sufficient?
Travis should mimic what works locally: does `./mvnw clean test -U -P spring` work locally?

Reply:
Is it even needed at all (apart from being explicit)? The profile is active by default and no other profiles are set.

Reply:
I think Travis must be setting another profile by default in the install script because all the builds failed until I added this. If we want green builds we have to do this IMO, or something like it, so can we just merge and move on?